### PR TITLE
HW done

### PR DESCRIPTION
**Pull request by asmaquraishi**

### CODE REVIEW

File: data-collections-exercises.md [hunk 1]
```diff
@@ -9,95 +9,40 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
...
 ```rb
-# Your answer here
+planeteers.delete("Captain Planet")
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]
```
Comment (ghadeer-x) [inline data-collections-exercises.md:30]: you can use the `+` operator to combine the two arrays together

File: data-collections-exercises.md [hunk 2]
```diff
@@ -9,95 +9,40 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
...
-# Your answer here
+ heroes.sort
+=> ["Black", "Blue", "Earth", "Fire", "Heart", "Pink", "Red", "Water", "Wind", "Yellow"]
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
-```
```
Comment (ghadeer-x) [inline data-collections-exercises.md:43]: what happens here you just skipped a whole section!

**Comment by ghadeer-x** [comment]
hey Asma great Job trying to do the hw 👏 , I wish you answer the form so I know how you feel about the hw I will mark it as complete but next time you need to try harder and communicate ok?

**Comment by asmaquraishi** [comment]
I'll try my best next time , thank you!

**Pull request closed without merging**